payments/swish/direct.md

```diff
@@ -0,0 +1,297 @@
+|          | └─➔&nbsp;`orderReference`             | `String(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|          | └─➔&nbsp;`subsite`                    | `String(40)`  | The subsite field can be used to perform split settlement on the payment. The subsites must be resolved with Swedbank Pay reconciliation before being used.                                                                                                                                                                                                                                                                                                                                                                                                               |
+|         | └─➔&nbsp;`msisdn`             | `String`  | Number will be prefilled on payment page, if valid.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|             | └─➔&nbsp;`ecomOnlyEnabled`             | `boolean`  | `true` if to only enable Swish on web based transactions.; otherwise `false` to also enable Swish transactions via in-app payments.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+
+{:.code-header}
+**Response**
+
+```http
+POST /psp/swish/payments HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "payment": {
+        "id": "/psp/swish/payments/20dfbcb9-587a-4ce9-e63e-08d519f1802f",
```

Comment:
```suggestion
        "id": "/psp/swish/payments/{{ page.paymentId }}",
```
Replace all occurrences of this guid with the constant.

Reply:
This still needs to be done, it seems. :)

Reply:
sorry, this is done now :)

Reply:
Have you forgotten to push? I can't see that it is done here on GitHub. :)